timelock-server/src/integ2Test/java/com/palantir/atlasdb/timelock/AbstractMultiNodePaxosTimeLockServerIntegrationTest.java

```diff
@@ -279,8 +263,8 @@ public void canCreateNewClientsDynamically() {
     }
 
     @Test
+    @Disabled
     public void lockRequestCanBlockForTheFullTimeout() {
```

Comment:
These tests, `waitForLocksRequestCanBlockForTheFullTimeout` and the current one, were not running previously. Now, when we attempt to run them, they fail. Should we proceed with the upgrade and leave a comment here to investigate later, or should we debug it now? I briefly tried to debug it but couldn't resolve the issue; however, I am willing to spend more time on it. (Although, I don't want to delay the completion of JUnit 5 migrations.)

Reply:
Probably worth digging into why they're failing, but I'd do it in a separate PR. Just add a comment to do so, and track in the migration issue and we should be good to merge this PR.